indra/newview/llfloatersidepanelcontainer.cpp

```diff
@@ -99,6 +99,10 @@ LLPanel* LLFloaterSidePanelContainer::openChildPanel(const std::string& panel_na
 	{
 	openFloater();
 	}
+	else if (!hasFocus())
+	{
+		setFocus(TRUE);
```

Comment:
This probably should do a bringtofront, not just setfocus

Reply:
It works as expected with  this change